doc/Reference/Reactive/state.md

```diff
@@ -104,6 +154,12 @@ This makes sure that you are working with the latest version of the data.
 
 This allows you to update the value only of the state.
 
+```csharp
+public static Task UpdateValue<T>(this IState<T> state, Func<T, T> updater, CancellationToken ct = default);
```

Comment:
Looking [here](https://github.com/unoplatform/uno.extensions/blob/main/src/Uno.Extensions.Reactive/Core/State.Extensions.cs), it seems `UpdateValue` is deprecated. We have `UpdateDataAsync` and `UpdateAsync`available, we should be documenting those two instead of `UpdateValue`

Reply:
And edit that current example code that is still using UpdateValue